### PR TITLE
Move Windows storage concepts to a new page

### PR DESCRIPTION
**Pull request by aravindhp**
Moving Windows storage concepts to a new page.

Part of https://github.com/kubernetes/website/issues/31428

/label refactor
/sig windows

### CODE REVIEW

**Comment by aravindhp** [comment]
/retitle Move Windows storage concepts to a new page

**Comment by marosset** [comment]
@aravindhp can you update the PR to target the dev-1.24 branch to match the rest of the PRs associated with https://github.com/kubernetes/website/issues/31428?

File: content/en/docs/concepts/storage/windows-storage.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+content_type: concept
+---
+
+<!-- overview -->
+
+This page provides an storage overview specific to the Windows operating system.
+
+<!-- body -->
+
+### Persistent storage {#storage}
```
Comment (marosset) [inline content/en/docs/concepts/storage/windows-storage.md:17]: Can you make this `##`?

**Comment by marosset** [comment]
Mostly LGTM,
thanks!

**Comment by aravindhp** [comment]
@marosset I have made the changes you requested

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-31900--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/windows-storage/
https://deploy-preview-31900--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/#windows-csi-proxy

**Comment by kbhawkey** [comment]
Ping
/sig storage

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -1271,6 +1271,21 @@ for more information.
 For more information on how to develop a CSI driver, refer to the
 [kubernetes-csi documentation](https://kubernetes-csi.github.io/docs/)
 
+#### Windows CSI proxy
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+CSI node plugins (especially those associated with persistent volumes exposed as
+either block devices or over a shared file-system) need to perform various privileged
+operations like scanning of disk devices, mounting of file systems, etc. These
```
Comment (kbhawkey) [inline content/en/docs/concepts/storage/volumes.md:1280]: nit: Try to avoid using `etc.`

**Comment by kbhawkey** [comment]
Changes LGTM

File: content/en/docs/concepts/storage/volumes.md [hunk 3]
```diff
@@ -1271,6 +1271,21 @@ for more information.
 For more information on how to develop a CSI driver, refer to the
 [kubernetes-csi documentation](https://kubernetes-csi.github.io/docs/)
 
+#### Windows CSI proxy
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```
Comment (msau42) [inline content/en/docs/concepts/storage/volumes.md:1276]: This is GA as of 1.22

File: content/en/docs/concepts/storage/windows-storage.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+of volumes, attaching/detaching a volume to/from a Kubernetes node and
+mounting/dismounting a volume to/from individual containers in a pod that needs to
+persist data.
+
+The code implementing these volume management actions for a specific storage back-end
+or protocol is shipped in the form of a Kubernetes volume
+[plugin](/docs/concepts/storage/volumes/#types-of-volumes).
+The following broad classes of Kubernetes volume plugins are supported on Windows:
+
+* [`FlexVolume plugins`](/docs/concepts/storage/volumes/#flexVolume)
```
Comment (msau42) [inline content/en/docs/concepts/storage/windows-storage.md:58]: FlexVolume is deprecated. We may want to add a notice here.

File: content/en/docs/concepts/storage/windows-storage.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- jayunit100
```
Comment (msau42) [inline content/en/docs/concepts/storage/windows-storage.md:3]: Please add @jingxu97 and @mauriciopoppe as reviewers

**Comment by aravindhp** [comment]
@msau42 I have addressed your comments. PTAL.

**Comment by marosset** [comment]
/hold
We want all the PRs that break up /modify https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/
See https://github.com/kubernetes/website/pull/31851#issuecomment-1079162797

**Comment by nate-double-u** [comment]
Does this need to target `dev-1.24`? 

/cc @marosset

**Pull request closed without merging**